msal/src/Microsoft.Identity.Client/Internal/PlatformPlugin.cs

```diff
@@ -32,24 +32,11 @@
 
 namespace Microsoft.Identity.Client.Internal
 {
-    internal static class PlatformPluginSwitch
-    {
-        static PlatformPluginSwitch()
-        {
-            DynamicallyLinkAssembly = true;
-        }
-
-        public static bool DynamicallyLinkAssembly { get; set; }
-    }
-
     internal static class PlatformPlugin
```

Comment:
>PlatformPlugin [](start = 26, length = 14)

I do not understand the intent of this class. Can you pls. help elaborate?

Reply:
it's used to get specific information about each platform, which we then use later to validate things, like the redirectUri